server/middleware/subscriptions.coffee

```diff
@@ -0,0 +1,23 @@
+errors = require '../commons/errors'
+wrap = require 'co-express'
+Prepaid = require '../models/Prepaid'
+log = require 'winston'
+
+subscribeWithPrepaidCode = wrap (req, res) ->
+  { ppc } = req.body
+  unless ppc and _.isString(ppc)
+    throw new errors.UnprocessableEntity('You must provide a valid prepaid code.')
+
+  # Check if code exists and has room for more redeemers
```

Comment:
By typo, you mean the comment no longer accurately describes the behavior? I don't see a grammatical error.